src/runtime/ltypes/ltypes.py

```diff
@@ -0,0 +1,4 @@
+i32 = []
+i64 = []
+f32 = []
+f64 = []
```

Comment:
```suggestion
f64 = []
c128 = []
```

Reply:
I merged it, as tests pass and this was an important PR. Can you submit this as a new PR?